docs/notebooks/getting_started/particles.ipynb

```diff
@@ -426,7 +426,7 @@
    "outputs": [],
    "source": [
     "argon = Particle(\"Ar 0+\")\n",
-    "argon.ionize(inplace=True)\n",
+    "argon = argon.ionize()\n",
```

Comment:
I need to revert this, since it's specifically showing off how to use the `inplace` parameter.